programmers/난이도별/level01.평균_구하기/HyeonJeong.java

```diff
@@ -0,0 +1,24 @@
+package 난이도별.level01.평균_구하기;
+
+public class HyeonJeong {
+    class Solution {
+        public double solution(int[] arr) {
+            float average;
+            float total = 0;
```

Comment:
```suggestion
            double average;
            double total = 0;
```
return type이 `double`이라서 그렇습니다!
`float` -> `double` 로 변경하면 잘 돼요~